xml/dm_registers.xml

```diff
@@ -198,7 +198,7 @@
             select. This hart is always part of the currently selected harts.
         </field>
         <field name="0" bits="5:4" access="R" reset="0" />
-        <field name="setresethaltreq" bits="3" access="W1" reset="0">
+        <field name="setresethaltreq" bits="3" access="W1" reset="-">
```

Comment:
for example, if it's already set here, you don't actually do anything when you W1 again.

Reply:
It's confusing that "set reset halt req" is a pretty clear "action" in your terminology, yet actually writing "set reset halt req" doesn't actually do anything...?

I'm not saying what youre writing is wrong or doesn't make sense, I just don't see that it's actually more clear than what's currently there.